<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>997</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(68, 68, 76);
background-color: rgb(68, 68, 76);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>Wyszukaj urządzenia szeregowe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(168, 13, 16);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>Połącz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>Rozłącz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>Pozycja lotek</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>Wizualizacja orientacji</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>Wykres</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>Test lotki</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>PL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background: rgb(204, 199, 172);
border: 2px solid rgb(0, 0, 0);
border-radius: 7px;</string>
            </property>
            <property name="text">
             <string>EN</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
background-color: rgba(204, 199, 172, 200);
border-radius: 7px;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Odczyty z sensorów</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="styleSheet">
              <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Estymowana orientacja rakiety</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="1">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>    Oś Z:  </string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>    Oś Y:  </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">text-align: right;</string>
                 </property>
                 <property name="text">
                  <string>    Oś X:  </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QGraphicsView" name="graphicsView_axis_icon">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="2" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>  Gyro Z:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>  Gyro Y:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>  Gyro X:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="3">
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="1" column="0">
                <widget class="QLabel" name="label_y_axis">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_z_axis">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_x_axis">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="8" column="3">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="3">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="6" column="3">
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="2" column="0">
                <widget class="QLabel" name="label_gyro_z">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_gyro_x">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_gyro_y">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="10" column="1">
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="1" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>   Acc Y:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>   Acc Z:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>   Acc X:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="10" column="3">
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="2" column="0">
                <widget class="QLabel" name="label_acc_z">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_acc_y">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_acc_x">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="6" column="0">
              <widget class="QGraphicsView" name="graphicsView_gyro_icon">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QGraphicsView" name="graphicsView_acc_icon">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="RocketRenderer" name="openGLWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsView">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(204, 199, 172, 200);
border-radius: 7px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="chart_frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RocketRenderer</class>
   <extends>QOpenGLWidget</extends>
   <header>rocketrenderer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>openGLWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>openGLWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>chart_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>chart_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>chart_frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>openGLWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
